CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+## []
+- Use of deprecated TextField after the upgrade of WTF to v3.0
+- Freeze to WTForms to version < 3
+
+## [4.41.1]
+### Fixed
+- General reports crash for variant annotations with same variant on other cases
```

Comment:
Hm, seems you have merged the hot fix into this already?!

Reply:
Yeah, it was the branch I've started this PR from, because I didn't realized the fix was released yet!